benchmarks/postprocess-base.py

```diff
@@ -101,6 +101,8 @@
       ## Total DOFs
       elif 'Global nodes' in line:
          data['num_unknowns']=int(line.split(':')[1])
+         if data['case']=='vector':
+            data['num_unknowns']/=3
```

Comment:
@jeremy is this what you meant?

Reply:
The other way around, I think. `num_unknowns` comes from the line giving `Global nodes`, and in the vector case `num_unknowns` is 3x larger than `Global nodes`, as `Global nodes` is `gsize/ncompu` and we really want `gsize`, the total number of DoFs we are solving for.

Reply:
I think I now got what you meant. Thanks!